streaming/vision/convert/ade20k.py

```diff
@@ -26,50 +26,50 @@ def parse_args() -> Namespace:
         '--in_root',
         type=str,
         required=True,
-        help='Directory path of the input dataset',
+        help='Local directory path of the input raw dataset',
     )
     args.add_argument(
         '--out_root',
         type=str,
         required=True,
-        help='Directory path to store the output dataset',
+        help='Directory path to store the output MDS shard files',
```

Comment:
is dataset

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/171/commits/ad49770b6c7f35b3e143cdf0fc6f5f699f7da917.